lte/gateway/python/magma/pipelined/openflow/flows.py

```diff
@@ -144,7 +145,8 @@ def add_flow(datapath, table, match, actions=None, instructions=None,
     # As 4G GTP tunnel, Register value is not used.
     # if use for default flow, 4G traffic is dropping.
     if ((goto_table != CLASSIFIER_NEXT_TABLE_NUM) and
-                     (goto_table != CLASSIFIER_INTERNAL_SAMPLING_FWD_TABLE_NUM)):
+              (goto_table != CLASSIFIER_INTERNAL_SAMPLING_FWD_TABLE_NUM) and
+              (goto_table != CLASSIFIER_INTERNAL_CONNTRACK_FWD_TABLE_NUM)):
```

Comment:
Yeah I don't understand the logic behind this, just use 
add_resubmit_next_service_flow

Reply:
If we do not put above condition then local scratch registers value will be placed with zero in actions, like below:
cookie=0x0, duration=158.456s, table=0, n_packets=0, n_bytes=0, priority=0,in_port=15579 actions=resubmit(,202),set_field:0->reg0,set_field:0->reg3